docs/Compliance.md

```diff
@@ -0,0 +1,124 @@
+# ATO and compliance considerations
+
+Every federal information system must be granted an Authority To Operate (ATO)
+by an Authorizing Official in order to go into production.  A good overview of
+the process can be found in this [excellent ATO overview](https://docs.google.com/presentation/d/1x-Bt8uyW-szHarglY57fFcC6PRCpUGFLe07uOvt93uk/edit#slide=id.g1f710bd7ce_0_847)
+and a more in-depth look can be found in [18F's before-you-ship docs](https://before-you-ship.18f.gov/ato/).  There
+are currently a few options for getting an ATO:
+* A [FedRAMP Tailored](https://tailored.fedramp.gov/) package.
+* A full [FedRAMP JAB ATO](https://www.fedramp.gov/jab-authorization/).
+* A GSA LATO (Lightweight ATO), which can be found by searching for "Lightweight
+  Security Authorization Guide" on 
+  [this page](https://insite.gsa.gov/topics/information-technology/security-and-privacy/it-security/it-security-procedural-guides)
+* Other agencies may have their own ATO process.
+
+What ATO you get depends on what your project needs.  If you are doing this in
```

Comment:
Let's remove the GSA option.

Reply:
I have it in there because it's actually one of the better documented procedures out there and gives a good idea of what hoops you will need to jump through.  However, I just realized that it's on insite, and thus not accessible outside of GSA.  Grr.  Thanks for catching that!